j-lawyer-client/src/de/costache/calendar/util/CalendarUtil.java

```diff
@@ -133,7 +133,7 @@ public static Collection<Date> getDates(final Date start, final Date end) {
 
     public static long getTotalSeconds(final Date date) {
         final Calendar c = CalendarUtil.getCalendar(date, false);
-        long seconds = c.get(Calendar.HOUR_OF_DAY) * 60l * 60l;
+        long seconds = c.get(Calendar.HOUR_OF_DAY) * 60L * 60L;

```

Comment:
just out of curiosity: does the case make a difference here?

Reply:
My IDE recommended changing it to the uppercase L :D
Seems to be a [readability recommendation](https://stackoverflow.com/questions/40953524/java-is-there-a-difference-between-l-and-l-lowercase-l-when-specifying-a-long#:~:text=No%20practical%20difference.,the%20corresponding%20Long%20wrapper%20type.&text=An%20integer%20literal%20is%20of,of%20type%20int%20(%C2%A74.2.)

Reply:
Makes perfect sense :-)

Going to merge this in a few minutes